File: Java/프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        String even = "";
        String odd ="";
        for(int i=0; i<num_list.length;i++){
            if(num_list[i]%2==0) {
                odd+=String.valueOf(num_list[i]);   
            }
            else {
                even+=String.valueOf(num_list[i]);   
            }
        }
        answer = (Integer.parseInt(odd))+(Integer.parseInt(even));
        return answer;
    }
} 

File: JavaScript/프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    var answer = 0;
    var odd = '';
    var even = '';
    for(var i of num_list){
        if(i%2==0){
            even += String(i);
        }
        else odd += String(i);
    }
    answer = Number(odd)+Number(even);
    return answer;
}